documentation/settings/topics/view-license.rst

```diff
@@ -30,23 +30,23 @@ The first three thresholds are presented visually on the page to help you gauge
 
 .. note:: There is no restriction on the number of users or locations that you can have. As of Hyperview 2.5, Monitored IP Addresses is no longer considered a license threshold.
 
-What counts as a "floor mounted asset"?
+What counts as an "asset"?
 ---------------------------------------
-From a licensing standpoint, a floor-mounted asset (or FMA) includes racks, UPS, PDUs, CRAC units, chillers, and generators. Your Hyperview pricing plan is based on the number of FMAs you wish to manage and monitor. Rack-mounted IT devices, such as servers, storage, network gear, patch panels, and switch ports are not counted or charged for, provided that they do not exceed the maximum number of sensors within a chosen plan. The default (free) plan supports up to 10 FMAs.
+An asset is any physical or virtual asset within your digital infrastructure that you discover (if network-connected) or manually add, manage, and/or monitor with Hyperview. These include operational technologies (racks, generators, UPS, PDUs, CRACs, CRAHs, RPPs, chillers), hardware (servers, storage, network devices, sensors/IoT, patch panel, and firewall), and virtual machines (VMs).
```

Comment:
I noticed you have a mix and plural and singular. For example, "servers" and "patch panel".